src/helpers/JSON-performance-report-parser.helper.js

```diff
@@ -0,0 +1,27 @@
+import fs from 'fs';
+
+const getReport = (fileName, path) => {
+  return JSON.parse(fs.readFileSync(`${path}/${fileName}`, 'utf8'));
+};
+
+class JSONPerformanceReportParser {
+  constructor(path) {
+    this.path = path;
+  }
+
+  parse(fileName) {
+    const reportFile = getReport(fileName, this.path);
+    const requests = reportFile.log.entries.map(item => ({
+      ttfb: item.timings.wait,
+      url: item.request.url
+    }));
+
+    if (requests.length > 0) {
+      return requests;
+    }
+
+    throw Error(`${fileName} contains incorrect data!`);
+  }
+}
+
+export const parser = (path = 'reports/performance') => new JSONPerformanceReportParser(path);
```

Comment:
Why such construction? Isn't it better to export just the clean class itself? 
`export JSONPerformanceReportParser`

Reply:
For now, I wanted to keep default `path` for performance testing. 
It is not configurable yet, I mean `path` where `.har` files are saved.

Reply:
In this case it does not really matter since this class has no real dependencies. The only problem I see is keeping the same path in case we start using it in multiple steps.

But ye, this can be changed to a normal constructor.

Reply:
@tgorskievo keep in mind that adjusting code to conform tests may not be a good idea :)

Reply:
You're right 👍